src/python/grapl-template-generator/grapl_template_generator/grapl-templates/rust-grpc-service/{{cookiecutter.project_slug}}/Cargo.lock

```diff
@@ -0,0 +1,5 @@
+# This file is automatically @generated by Cargo.
+# It is not intended for manual editing.
+[[package]]
+name = "snake_project_name"
+version = "0.1.0"
```

Comment:
We shouldn't be generating a lock file. Instead, we can use a `post_gen_project.sh` hook to run Cargo and let _it_ generate the file.